api/Document.json

```diff
@@ -1233,10 +1233,10 @@
               "version_added": null
             },
             "safari": {
-              "version_added": null
+              "version_added": "6"
```

Comment:
This is `document.contains` and doesn't match https://developer.mozilla.org/en-US/docs/Web/API/Node/contains. Is that entry wrong?

Reply:
The MDN URL for this feature is a 404, so I'm willing to bet this is a duplicate entry.  Although, the data should match, since Document inherits from Node...

Reply:
Opened #6841 for this!